src/main/java/jSearch/database/DatabaseConnectionHandler.java

```diff
@@ -49,23 +54,78 @@ public void close() {
         }
     }
 
-    public void deleteBranch(int branchId) {
+    public JobPosition[] getPositionsWithSalary(int minSalary) {
+        ArrayList<JobPosition> result = new ArrayList<>();
+
         try {
-            PreparedStatement ps = conn.prepareStatement("DELETE FROM branch WHERE branch_id = ?");
-            ps.setInt(1, branchId);
+            String query = "SELECT company_id, position_title, salary FROM job_position_compensation WHERE salary > ?";
```

Comment:
`job_position_compensation` is still being highlighted in red and showing up as "Unable to resolve table 'job_position_compensation' " in my intelliJ... but i tried running the query in psql shell and i have a feeling that this just has to do with main not compiling so i think once main is compiled it should work